backend/cfg/cfg_equivalence.ml

```diff
@@ -489,14 +494,17 @@ let check_basic_block : State.t -> Cfg.basic_block -> Cfg.basic_block -> unit =
   check_basic_instruction_list state location 0 expected.body result.body;
   check_terminator_instruction state location expected.terminator
     result.terminator;
-  State.add_label_sets_to_check state
-    (location ^ " (predecessors)")
-    expected.predecessors result.predecessors;
-  if not (Int.equal expected.trap_depth result.trap_depth)
-  then different location "trap depth";
-  State.add_label_sets_to_check state
-    (location ^ " (exceptional successors)")
-    expected.exns result.exns;
+  (* State.add_label_sets_to_check state (location ^ " (predecessors)")
```

Comment:
It cannot be re-enabled as-is, since the graphs are no longer
the same: the successors of unreachable blocks differ.
However, if the graphs are equivalent and well-formed, this
check was redundant anyway.